irodsbackend/static/irodsbackend/js/irodsbackend.js

```diff
@@ -254,11 +254,18 @@ $(document).ready(function() {
                         colSpan = '4';
                         icon = 'mdi:folder-open';
                         toolTip = 'Collection';
+
+                        var copyButton = $('<button>')
+                            .addClass('btn btn-secondary sodar-list-btn sodar-irods-copy-btn sodar-lz-copy-btn')
+                            .append($('<i>').addClass('iconify').attr('data-icon', 'mdi:console-line'))
```

Comment:
One thing I forgot to mentione before: here we should actually use `mdi:clipboard-text-multiple`. It is generally used as the "copy to clipboard" symbol in SODAR. In the case of multiple copy links we differentiate between the console etc, but here that's not needed.